opengcs/README.md

```diff
@@ -0,0 +1,34 @@
+
+# Open Guest Compute Service (opengcs) [![Build Status](https://github.com/microsoft/opengcs/actions/workflows/ci.yml/badge.svg)](https://github.com/microsoft/opengcs/actions/workflows/ci.yml)

```

Comment:
this won't work anymore :P did we update gh actions?

Reply:
Oh yeah I should've posted, I left only the README in the opengcs subdir as I didn't know what we'd want to do here. e.g. have a section on the main readme now that gives a quick blurb about opengcs or a link to where it used to be, or any other myriad of options haha

Reply:
Yep gh actions has a build_gcs step now